apps/studio/prisma/scripts/addCollectionIndexPage/index.ts

```diff
@@ -0,0 +1,86 @@
+import type { CollectionPagePageProps } from "@opengovsg/isomer-components"
+import {
+  COLLECTION_PAGE_DEFAULT_SORT_BY,
+  COLLECTION_PAGE_DEFAULT_SORT_DIRECTION,
+  ISOMER_USABLE_PAGE_LAYOUTS,
+} from "@opengovsg/isomer-components"
+import { TRPCError } from "@trpc/server"
+import { get } from "lodash"
+
+import { INDEX_PAGE_PERMALINK } from "~/constants/sitemap"
+import {
+  db,
+  jsonb,
+  ResourceState,
+  ResourceType,
+} from "~/server/modules/database"
+import { PG_ERROR_CODES } from "~/server/modules/database/constants"
+
+export const up = async () => {
+  const collectionsWithoutIndexPages = await db
```

Comment:
question: can I check when we create a collection inside Studio right now, does the index page get automatically created as well?